src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -154,7 +157,8 @@ public void setReceiver(String receiver) {
 
   @PostConstruct
   void postConstruct() {
-    userToCssStyle.put(userService.getUsername(), CSS_STYLE_SELF);
+    chatPrefs = preferencesService.getPreferences().getChatPrefs();
```

Comment:
Reading `ChatPrefs` only once and storing it becomes problematic if we reload the settings e.g. from disk or deserialize it (as discussed) since a new `ChatPrefs` instance will be created. This reference will then be stale.

Therefore we always have to call `preferencesService.getPreferences().***` whenever we access preferences.